misk-admin/web/tabs/web-actions/package.json

```diff
@@ -30,7 +30,10 @@
     "@misk/prettier": "0.4.0",
     "@misk/tslint": "0.4.0",
     "tslib": "2.3.0",
-    "@misk/cli": "0.4.0"
+    "@misk/cli": "0.4.0",
+    "webpack": "4.43.0",
+    "webpack-cli": "3.3.11",
+    "webpack-dev-server": "3.11.0"
```

Comment:
I had to add these packages to bring up a local dev server for the react code. Can remove these packages if there are others recommendations to run the client code.